src/Sulu/Bundle/AdminBundle/Resources/js/components/Table/Table.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+import React from 'react';
+import type {TableChildren} from './types';
+import tableStyles from './table.scss';
+
+type Props = {
+    /** Child nodes of the table */
+    children: TableChildren,
+};
+
+export default class Table extends React.PureComponent<Props> {
+    render() {
+        const {
+            children,
+        } = this.props;
+
+        return (
+            <div className={tableStyles.tableContainer}>
```

Comment:
Shouldn't only the body of the table be scrolling? Then we wouldn't need this container.